egs2/wsj0-2mix/enh1/enh.sh

```diff
@@ -0,0 +1,682 @@
+#!/bin/bash
```

Comment:
It would be better to be moved to https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE since it is used for both wsj0_2mix and wsj0_2mix_spatialized

Reply:
Done.